scripts/bootstrap.sh.in

```diff
@@ -94,41 +95,43 @@ install_salt_master() {
 }
 
 configure_salt() {
-        # TODO: Use a salt state and maybe other stuff
-        cat > "${SALT_MINION_FILE_CONF}" << EOF
-master: localhost
+    ${SALT_CALL} --retcode-passthrough state.apply metalk8s.saltmaster.minion_configured saltenv=metalk8s-@VERSION@ pillarenv=metalk8s-@VERSION@
+    rm -f "${SALT_MINION_FILE_CLIENT_LOCAL_CONF}"
+}
 
-# use new module.run format
-use_superseded:
-  - module.run
-EOF
-        rm "${SALT_MINION_FILE_CLIENT_LOCAL_CONF}"
-        cat > "${SALT_MASTER_FILE_CONF}" << EOF
-file_roots:
-  metalk8s-@VERSION@:
-    - /srv/scality/metalk8s-dev/salt
-pillar_roots:
-  metalk8s-@VERSION@:
-    - /srv/scality/metalk8s-dev/pillar
-peer:
-  .*:
-    - x509.sign_remote_certificate
-EOF
-        ${SYSTEMCTL} restart salt-master.service
-        ${SYSTEMCTL} enable salt-master.service
-        ${SYSTEMCTL} restart salt-minion.service
-        ${SYSTEMCTL} enable salt-minion.service
-        sleep 20
-        salt-key -A -y
+start_minion() {
```

Comment:
That's already done through the previous state, no?

Reply:
kept it for the 'enable' will rewrite it then